### PR TITLE
fix #1096 partially: Find a way to support custom Math expressions

### LINKED ISSUE #1096: Find a way to support custom Math expressions
Sample output of the english Wiktionary parsing:
```
<math> ERROR with \begin{align}\frac{\pi}{2} & = \prod_{n=1}^{\infty} \frac{ 4n^2 }{ 4n^2 - 1 } = \prod_{n=1}^{\infty} \left(\frac{2n}{2n-1} \cdot \frac{2n}{2n+1}\right) \\[6pt]& = \Big(\frac{2}{1} \cdot \frac{2}{3}\Big) \cdot \Big(\frac{4}{3} \cdot \frac{4}{5}\Big) \cdot \Big(\frac{6}{5} \cdot \frac{6}{7}\Big) \cdot \Big(\frac{8}{7} \cdot \frac{8}{9}\Big) \cdot \; \cdots \\\end{align} in [Wallis product]

<math> ERROR with f:\Z_2^n \to \Z_2 in [bent function]

<math> ERROR with \frac = \frac in [circle of Apollonius]

<math> ERROR with \and in [conjunction]

<math> ERROR with n\in\N in [elementary symmetric polynomial]

<math> ERROR with (x_1, x_2, \ldots, x_m, y)\in\R^{m+1} in [graph]

<math> ERROR with f: \R^m\rightarrow\R in [graph]

<math> ERROR with a_i \isin R in [group ring]

<math> ERROR with \R^3 in [halfway model]

<math> ERROR with \begin{align}\rho(g, h) (0,x_1,\ldots,x_k) &= g(x_1,\ldots,x_k) \\\rho(g, h) (y+1,x_1,\ldots,x_k) &= h(y,\rho(g, h) (y,x_1,\ldots,x_k),x_1,\ldots,x_k)\,\end{align} in [primitive recursion]

<math> ERROR with \isin in [supermodular]
```

I guess Mediawiki is using custom symbols somehow. We should find a way to handle them.

### PR DESCRIPTION
**Pull request by lasconic**
Fixes #

### CODE REVIEW

File: wikidict/utils.py [hunk 1]
```diff
@@ -475,7 +475,16 @@ def _convert_math(expr: str, packages: List[str] = []) -> str:
     from PIL import Image
     from sympy import preview
 
-    dvioptions = ["-T", "tight", "-z", "0", "-D 150", "-bg", "Transparent"]
+    expr = expr.replace("\\Z", "\\mathbb{Z}")
```
Comment (BoboTiG) [inline wikidict/utils.py:478]: Maybe add a comment to link the source for further additions?

**Review by BoboTiG — APPROVED** [review]

File: wikidict/utils.py [hunk 2]
```diff
@@ -420,9 +420,9 @@ def process_templates(word: str, text: str, locale: str) -> str:
 
         >>> process_templates("octonion", " <math>V^n</math>", "fr")  # doctest: +ELLIPSIS
         '<img style="height:100%;max-height:0.8em;width:auto;vertical-align:bottom" src="data:image/gif;base64,...'
-        >>> process_templates("test", r"<math>\R^n</math>", "fr")
-        <math> ERROR with \R^n in [test]
-        '\\R^n'
+        >>> process_templates("test", r"<math>\frac</math>", "fr")
```
Comment (BoboTiG) [inline wikidict/utils.py:423]: With that PR, there should be no more error, right?

Comment (lasconic) [reply wikidict/utils.py:423]: As explained in the issue,  `\frac` is not supported and gives an error.

Comment (BoboTiG) [reply wikidict/utils.py:423]: Ah indeed :)

**Pull request merged**